composeApp/src/commonMain/kotlin/features/favorites/presentation/model/FavoritesState.kt

```diff
@@ -0,0 +1,7 @@
+package features.favorites.presentation.model
+
+import core.presentation.model.ScreenState
+
+data class FavoritesState(
+    val state: ScreenState = ScreenState.Error
```

Comment:
Aqui nao seria Loading?
```suggestion
    val state: ScreenState = ScreenState.Loading
```

Reply:
deixei de qualquer jeito, vai mudar com a parte do use case kkk